app/helpers/api/bapi/getCatalogueItem.test.js

```diff
@@ -0,0 +1,25 @@
+import { getData } from 'buying-catalogue-library';
+import { getCatalogueItem } from './getCatalogueItem';
+import { solutionsApiUrl } from '../../../config';
```

Comment:
I think this is used throughout the project but found this a little confusing to start with, thought it was referring to a solutions endpoint. We might have used an alias in the additional service pages e.g. bapiUrl. This would be quite a refactor so not suggesting doing it now.

Reply:
shouldn't be too bad to change it all. However we have called the env var SOLUTIONS_API_URL 😢  we don't have to change the env var but nice if the line up.

Reply:
Yeah that's where we got stuck on the env var.